policy-controller/core/src/lib.rs

```diff
@@ -50,6 +50,7 @@ pub enum AuthorizationRef {
 #[derive(Clone, Debug, PartialEq, Eq, Hash)]
 pub enum InboundHttpRouteRef {
     Default(&'static str),
```

Comment:
do we only ever use the string "default" for the Default variant?  does it need to hold a str?